TimeIsMoney/AppDelegate.m

```diff
@@ -42,4 +52,22 @@ - (void)applicationWillTerminate:(UIApplication *)application {
     // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
 }
 
++(AppDelegate *) getAppDelegate
+{
+    return (AppDelegate*)[UIApplication sharedApplication].delegate;
+}
+
+-(void)saveSettings:(NSMutableArray*) completedTasks
+{
+    [self.localDefaults setObject:completedTasks forKey:self.savedUserTasks];
+    [self.localDefaults synchronize];
+}
+
+-(NSMutableArray*)getSettings
```

Comment:
seems like saveSettings and getSettings are slightly misnamed, they only save and return the completedTasks?